network/p2p/p2p_pubsub.go

```diff
@@ -53,16 +57,27 @@ func (n *p2pNetwork) Broadcast(msg *spectypes.SSVMessage) error {
 		return p2pprotocol.ErrNetworkIsNotReady
 	}
 
-	raw, err := commons.EncodeNetworkMsg(msg)
+	encodedMsg, err := commons.EncodeNetworkMsg(msg)
 	if err != nil {
 		return errors.Wrap(err, "could not decode msg")
 	}
 
+	if n.cfg.Network.RSAMessageFork(n.cfg.Network.Beacon.EstimatedCurrentEpoch()) {
```

Comment:
like the other comment, we might not need the function here, a simple if and storing the RSAMessageFork Epoch should be enough

Reply:
done